linux_docker_resources/Dockerfile

```diff
@@ -94,7 +94,7 @@ RUN if test \( ${UBUNTU_DISTRO} = jammy -a ${ROS_DISTRO} != humble \); then apt-
 # Iron uses python3-lttng, but then starting after Iron we switch to liblttng-ctl-dev.
 RUN if test \( ${UBUNTU_DISTRO} = jammy -a ${ROS_DISTRO} = iron \); then apt-get update && apt-get install --no-install-recommends -y \
   python3-lttng; fi
-RUN if test \( ${UBUNTU_DISTRO} = jammy -a ${ROS_DISTRO} = rolling \); then apt-get update && apt-get install --no-install-recommends -y \
+RUN if test \( ${UBUNTU_DISTRO} = noble -a ${ROS_DISTRO} = rolling \); then apt-get update && apt-get install --no-install-recommends -y \
```

Comment:
Here, we want to support Rolling on both `jammy` and `noble` for now.  So please update the conditional so either will work.